Eastern-Research-Group/csb-rebate-forms-app
# Feature/update 2022 crf status map

## Related Issues:
* CSBAPP-369

## Main Changes:
* Follow up to #456 – Adds "Withdrawn" status and updates logic for displaying "Close Out Not Approved" status for 2022 CRF.

## Steps To Test:
1. Navigate to the dashboard.
2. Work with the BAP team to ensure at least one of your 2022 CRF submissions has an internal status of "Withdrawn". Ensure "Withdrawn" is displayed for that submission.
3. Work with the BAP team to ensure at least one of your 2022 CRF submissions has an internal status of "Coordinator Denied". Ensure "Close Out Not Approved" is displayed for that submission.
